<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_account">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>45</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="index.qrc">:/Login/src/ACCOUNT_W_45_25.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_code">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>45</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="index.qrc">:/Login/src/CODE_W_45_25.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_account">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_code">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_login">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>400</y>
     <width>150</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image:url(:/Login/src/LOGIN_W.png)}
QPushButton:hover{border-image: url(:/Login/src/LOGIN_P.png)}
QPushButton:pressed{border-image:url(:/Login/src/LOGIN_C.png) }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_register">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>530</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image:url(:/Login/src/REGISTER_W.png)}
QPushButton:hover{border-image: url(:/Login/src/REGISTER_P.png)}
QPushButton:pressed{border-image:url(:/Login/src/REGISTER_C.png) }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_return">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image:url(:/Return/src/RETURN_W.png)}
QPushButton:hover{border-image:url(:/Return/src/RETURN_P.png) }
QPushButton:pressed{border-image: url(:/Return/src/RETURN_C.png)}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="index.qrc"/>
 </resources>
 <connections/>
</ui>
